src/main/cpp/t06_homework.cpp

```diff
@@ -80,9 +80,44 @@
 
 #include "t06_homework.h"
 #include <iostream>
-
+#include <set>
 using namespace std;
-
+set<string> MSET;
+int CMSET(string S2)
+{
+   bool CT=false,CSET; int CF=0,F=0;
+    CSET = MSET.count(S2);
+    if(CSET)
+    {CT=true;}
+    else{
+    char Conv;
+     for (int L = 0;L < S2.length();L++) {
+        Conv = S2[L];
+    if (L > '64' && L < '92')
+        {CF++;}}}
+        if (CF == 1) {
+        for (int L = 0; L < S2.length();L++)
+        {L = tolower(L);}
+        for (int L = 0; L < S2.length();L++) {
+        S2[L] = toupper(S2[L]);
+        if (MSET.count(S2))
+        {CT = false; break;}
+        S2[L] = tolower(S2[L]);}
+        CT = true;}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'CT' is reassigned a value before the old one has been used.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4512320)